### PR TITLE
Override previously defined extension points

### PR DESCRIPTION
**Pull request by cheungpat**
connects #58

### CODE REVIEW

File: skygear/registry.py [hunk 1]
```diff
@@ -54,37 +54,51 @@ def __init__(self):
...
+        in the registry, the existing param will be removed. An param
+        is a dictionary containing info of the declared extension point.
+        """
+        param_list = self.param_map[kind]
+
+        # Check existing param. Remove it if the param has the same name
+        # as the to-be-added param.
+        for i in range(len(param_list)):
+            if param_list[i].get('name') == param.get('name'):
+                del param_list[i]
```
Comment (rickmak) [inline skygear/registry.py]: I think it worth a `log.info`

Comment (rickmak) [inline skygear/registry.py]: May be debug?

**Comment by cheungpat** [comment]
updated the pr @rickmak

File: skygear/registry.py (hunk 1 above)
Comment (cheungpat) [inline skygear/registry.py]: will print log before/after calling this function

**Pull request merged**